go/src/koding/klient/machine/client/kite.go

```diff
@@ -23,29 +24,30 @@ func NewKiteBuilder(k *kite.Kite) *KiteBuilder {
 }
 
 // Ping uses kite network that stores kite's query string to ping the machine.
-func (kb *KiteBuilder) Ping(dynAddr DynamicAddrFunc) (Status, Addr, error) {
+func (kb *KiteBuilder) Ping(dynAddr DynamicAddrFunc) (machine.Status, machine.Addr, error) {
```

Comment:
non-related: the more I read `DynamicAddrFunc` the more I think `ResolveFunc` would fit better